socketio/examples/async.rs

```diff
@@ -14,8 +14,11 @@ async fn main() {
     let callback = |payload: Payload, socket: Client| {
         async move {
             match payload {
-                Payload::String(str) => println!("Received: {}", str),
+                Payload::Text(values) => println!("Received: {:?}", values),
```

Comment:
nit: maybe switch formatting to `{:#?}` to be consistent with how we format bytes in the next line?